scripts/get_timit_data.py

```diff
@@ -0,0 +1,147 @@
+# Copyright (c) 2019 NVIDIA Corporation
```

Comment:
Could you please remove NVIDIA's copyright - you wrote this, not us. If you want you can put your own copyright or put none at all.

Also, if you don't mention license, then project-wise license (which is Apache 2.0) is assumed.